pkg/app/pipedv1/plugin/kubernetes/deployment/determine.go

```diff
@@ -156,3 +156,16 @@ func findUpdatedWorkloads(olds, news []provider.Manifest) []workloadPair {
 	}
 	return pairs
 }
+
+func findConfigs(manifests []provider.Manifest) map[provider.ResourceKey]provider.Manifest {
```

Comment:
📝 Original method
https://github.com/pipe-cd/pipecd/blob/c2bf91d0e02165e1dbeff671d2f9d10d8a2bfda5/pkg/app/piped/planner/kubernetes/kubernetes.go#L388C1-L400C2

Reply:
Should we add a function comment like "findConfigs returns ConfigMap and Secret resources" to the head of this function? TBH, I don't actually like this implementation (even the original method) since we have to mention that this function returns both ConfigMap and Secret resources while its name makes the reader think it only returns config.

Reply:
@khanhtc1202 
I got your point. How about renaming this as `findConfigsAndSecrets`?

Reply:
That sounds nice 👍 @Warashi

Reply:
I renamed it and added a comment.
https://github.com/pipe-cd/pipecd/pull/5319/commits/dedd8fc5702e3ce0a8d3c71e27c63633d91ff8f9